application/datamanager/features/equity_bars.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Equity Bars Data Management
+    As a data manager application
```

Comment:
I think same comment from prior pull request of "datamanager" versus "data manager".